### PR TITLE
Use legacy builder to insert doc

### PR DESCRIPTION
**Pull request by texasRanger09**
Uses legacy builder to insert new docs. blocked til https://github.com/getditto/product/issues/159 is resolved.

### CODE REVIEW

File: DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt [hunk 1]
```diff
@@ -95,12 +95,10 @@ fun addToCollection(info: DittoHeartbeatInfo, config: DittoHeartbeatConfig, ditt
         "metaData" to metaData,
         "sdk" to info.sdk,
         "_schema" to info.schema,
-        "peerKey" to info.peerKey
+        "pk" to info.peerKey
```
Comment (bplattenburg) [inline DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt:98]: Is this change intentional? I think we settled on spelled out `peerKey` for the name here

File: DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt [hunk 2]
```diff
@@ -95,12 +95,10 @@ fun addToCollection(info: DittoHeartbeatInfo, config: DittoHeartbeatConfig, ditt
         "metaData" to metaData,
```
Comment (bplattenburg) [inline DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt:95]: I know this isn't changing as part of this PR, but I just noticed this is capitalized `metaData` here and `metadata` on the Swift version - we probably want to align those.

**Review by bplattenburg — CHANGES_REQUESTED** [review]

File: DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt (hunk 2 above)
Comment (texasRanger09) [reply DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt]: Updated with this commit: https://github.com/getditto/DittoSwiftTools/pull/98/commits/8921870cc94bf6cc7901fedeacbd27d132cb99fb

**Review by rajramsaroop — APPROVED** [review]
